### PR TITLE
fix: update accessiblity docs with correct content

### LINKED ISSUE #573: Breadcrumb Accessibility Tab updates and DAP violations
This issue is related to #491 
The following updates are needed to the Breadcrumb Accessibility Tab:
**1. Remove the first paragraph:** 
"The Breadcrumb React Carbon component has been tested against the latest W3C Web Content Accessibility Guidelines (WCAG) 2.1 Level A and AA success criteria and violations have been identified as high priority issues. This document will be updated when these accessibility issues are resolved."

**Note:** This paragraph is part of the Non-compliant template and should not be added to the completed documents. The non-compliant template is used when the document can not be completed due to open accessibility issues that block completion.

**2. Under the Heading "Accessibility Testing" add the following paragraph:**
Accessibility issues are tracked in the [Carbon Component GitHub repository](https://github.com/carbon-design-system/carbon/milestone/57).

**3. The following DAP violations are also found. Let me know if they need to be reported on a separate issue (see screenshot below):**
Screen Shot 2019-11-11 at 2 17 03 PM

### PR DESCRIPTION
**Pull request by aledavila**
Closes #573 #572 #574 

Fix a11y content on:

Accordion 
Breadcrumb
Checkbox

### CODE REVIEW

File: src/pages/components/accordion/accessibility.mdx [hunk 1]
```diff
@@ -131,19 +124,21 @@ This component has been validated to meet the [WCAG 2.0 AA](https://www.w3.org/T
...
         </StructuredListRow>
         <StructuredListRow>
           <StructuredListCell>
-            - macOS Mojave version 10.14.2 with VoiceOver
+            - macOS Mojave version 10.14.6 with VoiceOver
             <br />
-            - Safari version 12.0.2
-            <br />- Carbon 10 - React and vanilla
+            - Safari version 13.0.2
+            <br />- Carbon React Version 7.7.1
```
Comment (connor-leech) [inline src/pages/components/accordion/accessibility.mdx]:
```suggestion
            <br />- Carbon React version 7.7.1
```

File: src/pages/components/accordion/accessibility.mdx [hunk 2]
```diff
@@ -109,10 +102,10 @@ This component has been validated to meet the [WCAG 2.0 AA](https://www.w3.org/T
       <StructuredListBody>
         <StructuredListRow>
           <StructuredListCell>
-            - macOS Mojave version 10.14.2 with VoiceOver
+            - macOS Mojave version 10.14.6 with VoiceOver
             <br />
-            - Chrome version 71.0.3578.98 (Official Build) (64-bit)
-            <br />- Carbon 10 - React and vanilla
+            - Chrome version 77.0.3865.90
+            <br />- Carbon React Version 7.7.1
```
Comment (connor-leech) [inline src/pages/components/accordion/accessibility.mdx]:
```suggestion
            <br />- Carbon React version 7.7.1
```

File: src/pages/components/accordion/accessibility.mdx [hunk 3]
```diff
@@ -80,14 +72,15 @@ This component has been validated to meet the [WCAG 2.0 AA](https://www.w3.org/T
           <StructuredListCell>
             - macOS Mojave version 10.14.2 with VoiceOver
             <br />
-            - Chrome version 71.0.3578.98 (Official Build) (64-bit)
+            - Chrome Version Version 77.0.3865.90
```
Comment (connor-leech) [inline src/pages/components/accordion/accessibility.mdx]:
```suggestion
            - Chrome version 77.0.3865.90
```

**Comment by snidersd** [comment]
Editorial changes LGTM. Thx

**Pull request merged**